modules/ebi-metagenomics/interproscan/main.nf

```diff
@@ -0,0 +1,82 @@
+process INTERPROSCAN {
```

Comment:
Interproscan is problematic when it comes with the reference DB, the bioconda package downloads the db (which is massive >50G if my memory serves my well).

I suspect the container does something similar. In our pipelines we provide the reference db and have a patched version of interproscan.sh to support that.